apps/gallery/js/thumbnail_item.js

```diff
@@ -38,8 +38,7 @@ function ThumbnailItem(fileData) {
 ThumbnailItem.formatter = new navigator.mozL10n.DateTimeFormat();
 
 ThumbnailItem.prototype.localize = function() {
-  var date = new Date(this.data.date);
-  var imageCreatedAt = navigator.mozL10n.get('image-created-at');
-  var label = ThumbnailItem.formatter.localeFormat(date, imageCreatedAt);
+  var label = MediaUtils.shortDescription(
+    this.data.date, !!this.data.metadata.video);
```

Comment:
Nit: the preferred style in this app for multi-line function calls is to put the first argument on the same line as the open paren, and then align the others with that.

```
var label = MediaUtils.shortDescription(this.data.date,
                                        !!this.data.metadata.video);
```